pages/dashboard/index.js

```diff
@@ -61,6 +61,7 @@ const Dashboard = ({ user }) => {
 
   const [membership] = memberships?.data || [];
   const club = rawClub?.data || null;
+  const nationalTeam = user.national_team_interest == true;
```

Comment:
I wanted to add a link to the dashboard and thought this was probably the best way to do it - what do you think?

I have the link being yellow to start to try and separate it from the other links a little - make it feel a bit special and not as required. Not sure if that makes sense, but I'm pretty open to changing this as I'm sure you probs have opinions.

Long term it would be awesome to have a 'nationalTeamCard' on the page, that links to the profile to complete or displays the national team you're currently on the squad of... 😍

Reply:
For this specific line - comparing a boolean is true isn't the best way to do this, and certainly not double equals (triple equals all the way). Better to just use the value direct ie

```jsx
<ListItem
  mb={2}
  color={user?.national_team_interest ? 'keeperGreen' : 'seekerYellow'}
>
```

I'll need to have a look at it visually and check contrast is ok for a11y purposes. But otherwise this is fine for now 👍